src/bci_build/package.py

```diff
@@ -1852,10 +1852,8 @@ def _create_php_bci(
 """,
     )
     for ver, os_version in list(
-        product([15, 14], [OsVersion.SP5, OsVersion.TUMBLEWEED])
+        product([15, 14], [OsVersion.SP4, OsVersion.SP5, OsVersion.TUMBLEWEED])
     )
-    + [(pg_ver, OsVersion.TUMBLEWEED) for pg_ver in (13, 12)]
-    + [(pg_ver, OsVersion.SP4) for pg_ver in _POSTGRES_MAJOR_VERSIONS]
```

Comment:
then `_POSTGRES_MAJOR_VERSIONS` is unused, isn't it? or at least it needs updating to remove 10, 12, 13 ? (I thought I already did a PR for that, but apparently not)